services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/converters/attribute/LocalTimeAttributeConverterTest.java

```diff
@@ -0,0 +1,94 @@
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("-1")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterExceedHigherBoundTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("24:00:00")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterInvalidNanoSecondsTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("00:00:00.9999999999")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterNotAcceptLocalDateTimeTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("1988-05-21T00:12:00.000000001")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterNotAcceptInstantTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("1988-05-21T00:12:00.000000001Z")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterNotAcceptOffsetTimeTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("1988-05-21T00:12:00+01:00")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterNotAcceptZonedTimeTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("1988-05-21T00:12:00+01:00[Europe/Paris]")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterNotAcceptLocalDateTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("1988-05-21")
+                                                                       .toAttributeValue()));
+    }
+
+    @Test
+    public void LocalTimeAttributeConverterNotAcceptMonthDayTest() {
+        assertFails(() -> transformTo(converter, EnhancedAttributeValue.fromString("05-21")
+                                                                       .toAttributeValue()));
+    }
+
+    private <T> void verifyTransform(T objectToTransform, String attributeValueString) {
+        assertThat(transformFrom(converter, objectToTransform))
+            .isEqualTo(EnhancedAttributeValue.fromString(attributeValueString).toAttributeValue());
+        assertThat(transformTo(converter, EnhancedAttributeValue.fromString(attributeValueString).toAttributeValue()))
+            .isEqualTo(objectToTransform);
+    }
```

Comment:
This doesn't need to be parameterized since you're not passing in the attribute converter thus it's strongly typed to the class under test. Either drop the parameter <T> or move this into a shared place where all the tests can use it and pass in the converter to make it fully generic.